### PR TITLE
chore(CPlusPlus): created anagram

### PR DESCRIPTION
**Pull request by NegiAkash890**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

Added anagram.cpp

### CODE REVIEW

File: algorithms/CPlusPlus/Strings/anagram.cpp [hunk 1]
```diff
@@ -0,0 +1,29 @@
+//Description : To check whether a string is anagram or not.
```
Comment (UG-SEP) [inline algorithms/CPlusPlus/Strings/anagram.cpp:1]: better to tell the method you used 
hint: count character method

File: algorithms/CPlusPlus/Strings/anagram.cpp [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        temp[s1[i]]++;
+        temp[s2[i]]--;
+
+    }
+    for (int i = 0; i < 256; i++) {
+        if (temp[i] > 0) {
+            return false;
+        }
+    }
+    return true;
```
Comment (UG-SEP) [inline algorithms/CPlusPlus/Strings/anagram.cpp:18]: use few comment where the code is hard to understand so it's become user friendly program

**Review by UG-SEP — CHANGES_REQUESTED** [review]
Please kindly do the requested changes
Fell free to as any doubt related to this PR

**Comment by UG-SEP** [comment]
@NegiAkash890 any update???

**Review by UG-SEP — APPROVED** [review]
Thanks for contributing @NegiAkash890 
Great work😊
`The main thing which most impresses me was its clear cut user-friendly description of the program`

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**